### PR TITLE
Support auto_disarm_delay and disarm_kill_switch via MSP_MISC and MSP_SET_MISC

### PR DESCRIPTION
**Pull request by tricopterY**
cleanflight/cleanflight-configurator#77
Support for above enhancement. CF Configurator:development updated too.

### CODE REVIEW

**Comment by tricopterY** [comment]
Thanks for the heads up. Ok, I have to rework this file and all the stuff I have done for the Configurator cleanflight/cleanflight-configurator#130.
BTW - some people are asking for more settings like looptime etc to become available via CF Configurator.
What is your strategy eg new MSP_XXX, MSP_SET_XXX etc for each one. Please advise.

**Comment by tricopterY** [comment]
Ok, I just got the latest release-candidate and the guidelines stipulate the uses. Got it.

**Comment by hydra** [comment]
great.  do you want to update this PR and the configurator PR and maybe also address #630 with a separate PR too?

File: src/main/io/serial_msp.c [hunk 1]
```diff
@@ -980,7 +980,7 @@ static bool processOutCommand(uint8_t cmdMSP)
         }
         break;
     case MSP_MISC:
-        headSerialReply(2 * 6 + 4 + 2 + 4);
+        headSerialReply(2 * 6 + 4 + 2 + 4 + 2);//24 bytes allows for disarm settings
```
Comment (hydra) [inline src/main/io/serial_msp.c]: no need to add comment here, the values can be easily correlated with the code.

**Comment by tricopterY** [comment]
I will do the lot. Should I put the looptime in the setup tab just under the Instrument panels, It may be more appropriate there - with option to pick from a spinner UI with steps of 500 and limits 0 - 9000.

**Pull request closed without merging**